generator/scrapper/extract/class.ts

```diff
@@ -18,6 +18,9 @@ export function extractClass(page: WikiPage): WikiFunctionCollection {
             address: page.address,
         };
     } else {
+        if (!markupObj.panel) {
+            return null;
```

Comment:
Is it possible to still return something here? You can extract the description from the markupObj and set the parent to null. You could also return something different than WikiFunctionCollection if it fits pixelvis_handle better.